### DISCUSSION TITLE
Mattermost

### DISCUSSION THREAD

**Discussion by ep9io** [apache/hop#4728] [Q&A] [answered]
My first discussion here :-).   So ticket related stuff within the actual ticket, general Q&A here?   Avoid using mattermost if possible then?

**Comment by bamaer** [comment, marked as answer]
yes, bug reports, tasks and feature requests in issues, general discussions, Q&A etc here. 
Mattermost will be going away after this year because it's not on the Apache Hop subdomain, not indexable and the content is often scattered throughout tons of simultaneous discussions

**Comment by ep9io** [comment]
Got it.  This is a test reply directly from email.
